poppy/sub_sampled_optics.py

```diff
@@ -0,0 +1,253 @@
+import numpy as np
+import poppy
+import matplotlib.pyplot as plt
+import astropy.units as u
+
+from .poppy_core import OpticalElement, Detector, Wavefront, PlaneType, _PUPIL, _IMAGE, _RADIANStoARCSEC
+from .optics import CircularAperture
+from .utils import measure_centroid
+
+import logging
+_log = logging.getLogger('poppy')
+
+class subapertures(poppy.OpticalElement):
```

Comment:
Also, it's generally recommended to capitalize class names for consistency (e.g. `Subapertures` instead of `subapertures`)